server/migrations/20220427204420-changeAtVersionColumn.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
@howard-e @alflennik This is the last migration file. For some reason changing the column name from `atVersion` to `name` isn't working. Trying to figure this one out, but could use some help

Reply:
What is the error that's thrown?

You could try removing the `atVersion` column then adding the `name` column in the migration if all else fails

Reply:
Okay seems that's solved! 👍🏽